tests/tests_post_binary_metadata.py

```diff
@@ -0,0 +1,440 @@
+import unittest
+import responses
+import json
+import io
+from tests.endpoint_testing import BaseTestCases
+
+
+class PostBinaryMetadataMarketPage(BaseTestCases.BaseAppTesting):
```

Comment:
There are no tests for removing screenshots?
We could also have a test for removing icon (even if we don't have UI for it yet).

I'm not sure if I seen a test for when images fail to upload (icon is wrong size or screenshots in wrong format).

Also, I think we still have a bug that if you add twice same image (which is possible in our UI) upload will fail. Maybe we could test this specific edge case (and fix it). But not sure if it needs to be part of this PR if it's too big.